vdp/pipeline/v1beta/integration.proto

```diff
@@ -0,0 +1,242 @@
+syntax = "proto3";
+
+package vdp.pipeline.v1beta;
+
+// Google API
+import "google/api/field_behavior.proto";
+// Protocol Buffers Well-Known types
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+// VDP definitions
+import "vdp/pipeline/v1beta/common.proto";
+
+// Connection contains the parameters to communicate with a 3rd party app. A
+// component may reference a connection in their setup. One connection may be
+// used by several components and pipelines.
+message Connection {
```

Comment:
ℹ️ I respected the names in the product wireframe for the entities in the contracts. However, as mentioned there, there's a high overlap between the Integration and the ComponentDefinition entities (in particular, its Setup property in the ComponentSpecification field).

We might want to challenge these names, here are some suggestions for Connection and Integration, respectively:
 - Integration and IntegrationSchema.
 - Connection and ConnectionSchema. 
 - Integration and [Integrated]App.
 - Connection and App.